CONTRIBUTING.md

```diff
@@ -4,9 +4,11 @@
 For the most up-to-date instructions see the github actions [test.yml workflow](./github/workflows.test.yml)
 
 1. Install conda and mamba
-1. mamba create -n test pip pytest-cov pytest-xdist pip
-1. conda activate test
-1. python -m pip install -r requirements.txt
-1. python -m pip install -r requirements-dev.txt
-1. pip install -e . --no-deps --force-reinstall
-1. pytest -n auto -vrsx --cov=conda_lock tests
\ No newline at end of file
+1. `mamba create -n test pip pytest-cov pytest-xdist pip`
+1. `conda activate test`
+1. `python -m pip install -r requirements.txt`
+1. `python -m pip install -r requirements-dev.txt`
+1. `pip install -e . --no-deps --force-reinstall`
+1. `pytest -n auto -vrsx --cov=conda_lock tests`
+1. `pre-commit`
```

Comment:
Shouldn't this be pre-commit install?

Reply:
Maybe? I thought it installed if you just ran pre-commit

Reply:
Nope that just runs the default config.